governance.md

```diff
@@ -8,12 +8,18 @@ https://github.com/pangeo-data/governance
 
 ## The Project
 
-The Pangeo Project (The Project) is an open source software project. The goal 
-of The Project is to develop open source software and related technology for the 
-analysis of large scientific datasets. The Project endeavors to extend the 
-broader scientific software ecosystem.  The Software developed by The Project is
-released under the BSD (or similar) open source license, developed openly and 
-hosted in public GitHub repositories under the Pangeo-data GitHub organization.
+The Pangeo Project (The Project) is first and foremost a community of people
+working collaboratively to develop software and infrastructure to enable Big Data
```

Comment:
I'm worried about the language here being so geoscience-specific.  It seems to me that Pangeo has expanded beyond the geosciences with neuro.pangeo.io and astro.pangeo.io.  I admit that the "Pangeo" name suggests a strong geoscience connection, which it has, but I'm worried that maybe Pangeo has become something more...?

Reply:
The wording is indeed geoscience specific, since it comes from the original mission statement. There (and in the name Pangeo) we intended for it to be geoscience specific. For example, we discussed originally how while xarray or dask are not geoscience-specific, we might be able to make packages that supply the geoscience-specific functionality we need from packages like xarray or dask, and contribute back to those projects. This also comes across strongly when you look at the [homepage](http://pangeo.io/).

Of course, that was then, and this is now. Has the purpose of the organization changed? I could see astronomy falling under a wide definition of geoscience, but I'm surprised to hear that we're doing neuroscience now. Reading #399, it makes more sense, and I would say the purpose hasn't changed. The reason neuro.pangeo.io makes sense is because it's meant to develop tools central to geoscience (xarray, dask, jupyterhub, and hdf-related tools). @arokem is "developing software and infrastructure to enable Big Data geoscience research" even if he is working on neuroscience. I'd point out that we are "leveraging the existing expertise outside of the geoscience community" in this case.

I'd worry that widening the mission statement to "developing big data tools for science" is far too broad.

Reply:
@rabernat @jhamman @chiaral It would be good to hear others weigh in on this.

Reply:
I've gone both ways on this one. My current position is that pangeo needs to remain geoscience focused and work with other communities of research to extend the pangeo-model. With that in mind, I'm +1 on this edit.

Reply:
Could we add a little sentence saying that the first mission of Pangeo is geoscience, but that the community is also trying to improve things for other sciences?